authors/settings.py

```diff
@@ -137,7 +137,7 @@
     'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
     'NON_FIELD_ERRORS_KEY': 'error',
 
-    'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
-    ),
+    # 'DEFAULT_AUTHENTICATION_CLASSES': (
```

Comment:
why is this code commented out

Reply:
This piece of code breaks the app. Had commented it out to get feedback whether or not to get rid of it.

Reply:
@ja-odur, @felixkiryowa will clean up this code when writing the tests. I've talked to him and he has agreed.